ibis/expr/types.py

```diff
@@ -1506,7 +1509,34 @@ def root_tables(self):
 
 
 class ListExpr(ColumnExpr, AnyValue):
```

Comment:
I expanded the implementation of this class to allow using it for lists of ibis expressions throughout the library